ocs_ci/ocs/ui/base_ui.py

```diff
@@ -830,6 +830,8 @@ def login_ui(console_url=None):
     wait = WebDriverWait(driver, 60)
     driver.maximize_window()
     driver.get(console_url)
+    # Validate proceeding to the login console before taking any action:
+    proceed_to_login_console(driver)
```

Comment:
1. This additional login page is applicable only for ACM login, not for OCP login.
2. Is it applicable for all the ACM versions?

Reply:
The login function is common across the UI deployment + ACM automation. So that's why I thought it is a good placement here in case we hit this page. @AviadP also mentioned to me that he saw this once in regular UI deployment.

I can only confirm that I faced this in ACM V2.4.4

Reply:
I disagree, I have never seen this additional page while logging into the OCP console.
This needs to be only used for ACM login. Not for OCP. You can validate the same locally or by running a PR validation job. I don't think it will pass.
Though I can confirm it appears for RHACM 2.5 (Downstream).